concepts/Common/Security Considerations/20 HTML Encoding/20 Encode Template Data.md

```diff
@@ -1,4 +1,4 @@
-Angular, Vue, and React always encode values interpolated in templates. With other frameworks and libraries, use a third-party sanitizing tool as follows:
+To encode values interpolated in templates, use a third-party sanitizing tool as follows:
```

Comment:
What do you mean by `interpolated`?

Reply:
I think this expression is misleading. So I have 
removed it